prestoadmin/config.py

```diff
@@ -71,6 +71,26 @@ def json_to_string(conf):
     return json.dumps(conf, indent=4, separators=(',', ':'))
 
 
+def write_conf_to_file(value, path):
```

Comment:
Can we please get a comment that notes that this only works for flat configs so that when somebody tries to use it for something hierarchical, they know that they need to add functionality?